### ISSUE TITLE
BotSpawns removed?

### ISSUE THREAD

**Issue by ssamjh** [jaredballou/insurgency-sourcemod#12]
I cannot find the file to download?
Has it been removed?

**Comment by jaredballou** [comment]
Yes, the latest Insurgency 2.x branch that was released broke the old spawn behavior and they are basically merging my ideas into the game. I don't have the time to fix it, when the upstream game is changing so dramatically and so frequently.

**Comment by hyperbolic2346** [comment]
Do you have a link to up-to-date documentation on the servers? All I can find are random pieces of information around and some of it works and some of it doesn't. I'd like to know about the settings to do some of the things that this module used to do.

**Comment by jaredballou** [comment]
So I have started working on this plugin again. I split out the respawn stuff entirely, and just do spawn management. I'm hoping to just start off by having it spawn bots away from players, and not pile the entire wave from thin air. If you're interested in helping out, let me know.
